Userland/Libraries/LibGUI/Window.cpp

```diff
@@ -499,7 +499,7 @@ void Window::propagate_shortcuts_up_to_application(KeyEvent& event, Widget* widg
 
     if (!action)
         action = action_for_shortcut(shortcut);
-    if (!action)
+    if (!action && limit == PropagationLimit::Application)
```

Comment:
technically this should probably be
```suggestion
    if (!action && limit >= PropagationLimit::Application)
```
and the previous one would be then
```cpp
    if (!action && limit >= PropagationLimit::CurrentWindow)
```